packages/app/src/index-storage/observers/index-storage-loading.observer.test.ts

```diff
@@ -36,12 +37,18 @@ describe("Method: observe", () => {
   })
 
   test("calls the `DeviceService.request` and `IndexStorageService.loadIndex` methods when `DeviceServiceEvent.DeviceUnlocked` has been emitted", async () => {
-    deviceManager.device.request = jest.fn().mockResolvedValueOnce({
-      data: {
+    // deviceManager.device.request = jest.fn().mockResolvedValueOnce({
+    //   data: {
+    //     deviceToken: "1234567890",
+    //     serialNumber: "0000000000",
+    //   },
+    // })
```

Comment:
to remove? 🤔

Reply:
Removed in https://github.com/mudita/mudita-center/pull/1078/commits/5d1aa7079b4c1c5da6ad60388501daeb034f7b20